openjdk.test.load/src/test.load/net/adoptopenjdk/stf/MauveMultiThreadLoadTest.java

```diff
@@ -60,11 +60,12 @@ public void execute(StfCoreExtension test) throws StfException {
 			inventoryFile = "/openjdk.test.load/config/inventories/mauve/mauve_multiThread_osx_openj9_jdk8.xml";
 		}
 		
-		/* Temporarily exclude DecimalFormat.setGroupingSize from openj9 jdk14 
+		/* Temporarily exclude DecimalFormat.setGroupingSize from openj9 jdk14 and jdk15
 			Ref: https://github.com/eclipse/openj9/issues/8086
+				 https://github.com/eclipse/openj9/issues/8620
 		 */
 		if (jvm.isIBMJvm() 
-			&& jvm.getJavaVersion() == 14) {
+			&& (jvm.getJavaVersion() == 14 || jvm.getJavaVersion() == 15)) {
```

Comment:
How about `jvm.getJavaVersion() >= 14` to exclude future versions?

Reply:
Sure. Updated PR.